charts/lagoon-core/Chart.yaml

```diff
@@ -21,7 +21,7 @@ type: application
 # time you make changes to the chart and its templates, including the app
 # version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.54.2
+version: 1.54.3
```

Comment:
Can you revert this please, we'll handle bumping the version as we close out the release cycle.